index.html

```diff
@@ -84,10 +84,55 @@ <h5>Multi-Post Stories</h5>
           >
         </div>
       </div>
+      <div class="multi-post-box-wrapper d-flex Menu-mobile-after">
+        <div class="multi-post-box">
+          <div class="multi-post d-flex">
+            <h1>Multi Post Stories</h1>
+            <img
+              src="pictures/detail/Icon.svg"
+              class="multi-post-icon"
+              alt=""
+            />
+          </div>
```

Comment:
The text is overlapping with the button on the mobile popup window
![wrong](https://user-images.githubusercontent.com/58376020/127697129-3c430d0c-a885-4eb3-8f39-f19b0930dafd.png)